irssi/README.md

```diff
@@ -0,0 +1,29 @@
+Dockerfile-Fedora-irssi
+=======================
+
+Fedora Dockerfile for [irssi](https://irssi.org/).
+
+To Run the Container:
+---------------------
+
+```
+# docker run -it fedora/irssi
+```
+
+This will start the irssi client
+
+To Build:
+---------
+
+Copy the sources down and perform Build
+
+```
+# docker build -t <username>/irssi .
+```
+
+To Run:
+-------
+
+```
+# docker run -it <username>/irssi
```

Comment:
@scollier I was thinking if we could add irssi to Fedora docker hub itself, then we could simply use `fedora/irssi` only.
`<username>/irssi` is for if people want to build themselves.
Yes we want to run it in interactive mode. Our purpose is to use irssi on atomic host. so more precisely this is for atomic users :).